### PR TITLE
Feature/edit players page

### PR DESCRIPTION
**Pull request by leachtucker**
# Description
Parents need a new place to add & update their childrens' information.

Fixes #

- What work was done?
- Why was this work done?
- What feature / user story is it for?
_As a parent, I want to be able to add/edit my children's profiles on the fly, so that I can make changes if anything occurs._
- Any relevant links or images / screenshots

## Type of change

- [X] New feature (non-breaking change which adds functionality)

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [X] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [x] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My code has been reviewed by at least one peer
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

### CODE REVIEW

**Comment by leachtucker** [comment]
[As a parent, I want to be able to add/edit my children's profiles on the fly, so that I can make changes if anything occurs.](https://trello.com/c/nZKSDIFm/55-as-a-parent-i-want-to-be-able-to-add-edit-my-childrens-profiles-on-the-fly-so-that-i-can-make-changes-if-anything-occurs)

File: src/api/index.js [hunk 1]
```diff
@@ -65,6 +65,9 @@ const apiAuthPost = (endpoint, body, authHeader) => {
 const apiAuthPut = (endpoint, body, authHeader) => {
   return axios.put(`${getApiUrl()}${endpoint}`, body, { headers: authHeader });
 };
+const apiAuthDelete = (endpoint, authHeader) => {
```
Comment (leachtucker) [inline src/api/index.js:68]: apiAuth methods for all of the CRUD operations except for delete. I implemented that here.

File: src/index.js [hunk 2]
```diff
@@ -156,6 +157,12 @@ function App() {
             <AddChild LoadingComponent={ParentLoadingComponent} />
           )}
         />
+        <SecureRoute
+          path="/parent/edit-players"
```
Comment (leachtucker) [inline src/index.js:167]: This feature has been added to the web app under the '/parent/edit-players' path

**Review by StefxM — DISMISSED** [review]
looks good!

**Review by cearc-sara — APPROVED** [review]
This looks great, awesome job

**Review by MosharrafMusa — APPROVED** [review]
look good

**Review by StefxM — APPROVED** [review]
looks greatt

**Pull request merged**